css-positioning/absolute.css

```diff
@@ -0,0 +1,47 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  left: 220px;
```

Comment:
This is a rather large movement. Since this rests against the right edge, you should use the `right` box offset property